edit_course/operations/configure.py

```diff
@@ -509,7 +512,9 @@ def get_target_category(category, course=None, course_url=None):
             course_instance = CourseInstance.objects.get(url=instance_slug,
                                                          course__url=course_slug)
         except ObjectDoesNotExist:
-            return None, _('No course found with URL "{}"'.format(course_url))
+            return None, _('No course found with URL "{}"; \
```

Comment:
no need for escape and there is wrong character ';' instead of ','.

Reply:
Escapes are used to split that one long string to multiple lines. Not using them would cause End Of Line errors.

I wanted to add slugs as debug-aid to the error message. I though that ';' would keep the "main message" separated and provide the slugs as additional information. '.' might be better...